app/models/concerns/foreman_host_reports/host_extensions.rb

```diff
@@ -4,13 +4,27 @@ module HostExtensions
 
     included do
       has_many :host_reports, foreign_key: :host_id, class_name: 'HostReport', inverse_of: :host, dependent: :destroy
-      has_one :last_host_report_object_any_format, -> { order("#{HostReport.table_name}.reported_at DESC, #{HostReport.table_name}.id").limit(1) }, foreign_key: :host_id, class_name: 'HostReport', inverse_of: :host, dependent: :delete
+      # rubocop:disable Rails/HasManyOrHasOneDependent
+      has_one :last_host_report_object_any_format, -> { order("#{HostReport.table_name}.reported_at DESC, #{HostReport.table_name}.id").limit(1) }, foreign_key: :host_id, class_name: 'HostReport', inverse_of: :host
```

Comment:
Why do we even need this then if we have an explicit `last_host_report_object` "relation" defined below? Also, AFAIU, this is being used only here: https://github.com/theforeman/foreman_host_reports/blob/42495533ef2e47c8b5cb13cc0ad482cdd5bac73c/app/helpers/concerns/foreman_host_reports/hosts_helper_extensions.rb#L12

Reply:
Good point, that is really unnecessary.